website/project/model.py

```diff
@@ -2158,8 +2159,13 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             foreign_user=foreign_user,
             params=params,
         )
+
         if log_date:
             log.date = log_date
+
+        self.date_modified = log.date.replace(tzinfo=None)
```

Comment:
Is there any reason to do this here, rather than in `.save`? No specific examples come to mind (maybe user merging), but it's possible that there's an action that changes and saves a node without adding a log.